modules/apps/commerce/commerce-service/src/main/java/com/liferay/commerce/service/impl/CommerceOrderItemLocalServiceImpl.java

```diff
@@ -2641,6 +2642,51 @@ private void _validate(
 		}
 	}
 
+	private void _validateBundleInformation(
+			CPDefinitionOptionRel cpDefinitionOptionRel,
+			List<CPDefinitionOptionValueRel> cpDefinitionOptionValueRels,
+			JSONArray optionJSONArray)
+		throws PortalException {
+
+		for (CPDefinitionOptionValueRel cpDefinitionOptionValueRel :
+				cpDefinitionOptionValueRels) {
+
+			String linkedCPInstanceUUID =
+				cpDefinitionOptionValueRel.getCPInstanceUuid();
+
+			if (Validator.isNotNull(linkedCPInstanceUUID)) {
+				CPInstance linkedCPInstance =
```

Comment:
Just to make it clear, this is the CPInstance for the linked product and is therefore not the same as the CPInstance we have from earlier.